ogr/services/github.py

```diff
@@ -317,6 +318,13 @@ def get_sha_from_tag(self, tag_name: str) -> str:
                 return tag.commit.sha
         return ""
 
+    def get_tag_from_tag_name(self, tag_name: str) -> Optional[GitTag]:
+        all_tags = self.github_repo.get_tags()
+        for tag in all_tags:
+            if tag.name == tag_name:
+                return GitTag(tag.name, tag.commit.sha)
```

Comment:
Personally, I like the more explicit way:

```suggestion
                return GitTag(name=tag.name, commit_sha=tag.commit.sha)
```